### ISSUE TITLE
Addition to GO_annotation model

### ISSUE THREAD

**Issue by vanaukenk** [WormBase/wormbase-pipeline#186]
Hi,

I'd like to propose an additional tag for the GO_annotation model: Noctua_model_id

We are in the process of migrating our manual GO annotations from the Caltech OA and UniProt's Protein2GO to Noctua. 

Once the migration is complete, GO will be the source of WB manual annotations, so we won't want to send them back to the GOC in our GAF.

The Noctua_model_id tag would be populated with each unique model id with the idea that we could use this tag as a filter for what we export to the GAF we produce with each release.  There might be other uses for this tag in WB, e.g. linking out to the Noctua representation of any set of annotations.

I'm happy to discuss further if need be.  There's going to be some churn on the GO data pipeline in the coming months, so we'll want to make sure we have a good roadmap.

Thanks.

**Comment by Paul-Davis** [comment]
Thanks @vanaukenk I only just spotted this. No issues with the new tag, do you think it would be good to add the standard external database xref tags to the GO_annotation model like so?

```
Noctua_model_id ?Text
DB_info Database ?Database ?Database_field ?Text
```

**Comment by Paul-Davis** [comment]
closed by commit 30ed325b5718ab444bee7c5cd55ec2616c489b64
